node/parallel-dev/src/chain_spec.rs

```diff
@@ -187,12 +187,12 @@ fn testnet_genesis(
                 CurrencyId::USDT,
                 CurrencyId::xDOT,
             ],
-            borrow_index: Rate::one(),                                  // 1
-            exchange_rate: Rate::saturating_from_rational(2, 100),      // 0.02
-            base_rate_per_year: Rate::saturating_from_rational(2, 100), // 0.02
-            multiplier_per_year: Multiplier::saturating_from_rational(1, 10), // 0.1
-            jump_multiplier_per_year: Multiplier::saturating_from_rational(11, 10), // 1.1
-            kink: Ratio::from_percent(80),                              // 0.8
+            borrow_index: Rate::one(),                             // 1
+            exchange_rate: Rate::saturating_from_rational(2, 100), // 0.02
+            base_rate: Rate::saturating_from_rational(2, 100),     // 2%
+            kink_rate: Rate::saturating_from_rational(10, 100),    // 10%
```

Comment:
`kink_rate` isn't a good name as this value is between `0` and `1`, ratio is better